libraries/AP_ExternalAHRS/AP_ExternalAHRS.h

```diff
@@ -121,6 +121,9 @@ class AP_ExternalAHRS {
     bool get_accel(Vector3f &accel);
     void send_status_report(class GCS_MAVLINK &link) const;
     bool get_variances(float &velVar, float &posVar, float &hgtVar, Vector3f &magVar, float &tasVar) const;
+    void set_data_send_disable(bool disable);
+    void set_gnss_disable(bool disable);
```

Comment:
```suggestion
    void set_gnss_disable(bool disable);
```

Does setting this to false re-enable? If so, thoughts on calling it `set_gnss_state(bool enabled)`?

Reply:
Yes, I renamed them to

```cpp
void set_data_sending_state(bool enabled);
void set_gnss_state(bool enabled)
```